crates/ide/src/references.rs

```diff
@@ -112,6 +114,26 @@ pub(crate) fn find_all_refs(
     }
 }
 
+fn retain_import_usages(usages: &mut UsageSearchResult, sema: &Semantics<'_, RootDatabase>) {
```

Comment:
The name is inverted here, this retains anything but imports, so `filter_import_references` would probably fit better

Reply:
I was inspired by
```
    /// Filter out all non-literal usages for adt-defs
    fn retain_adt_literal_usages(
```

:)

Reply:
Yes, but there it actually keeps only the adt literals, which is what `retain` means, in your case you are not retaining imports though, you are retaining anything but imports

Reply:
Yep. May be I was tired a bit